docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+services:
+  web:
+    build: .
+    ports:
+      - "8080:8080"
+    environment:
+      - JAVA_OPTS=-Dspring.data.mongodb.uri=mongodb://db:27017/session-service
```

Comment:
@inodb looks good. but what if MongoDB is running on a different server? should we make it configurable

Reply:
@kalletlak thanks! What part would you like to see configurable? One can change the `JAVA_OPTS` env variable to point to a mongo database on a different server if so desired

Reply:
that should work. thanks!